task/src/lastpencil/PencilGame.java

```diff
@@ -3,4 +3,113 @@
 import java.util.Random;
 import java.util.Scanner;
 
-//Class for Pencil Game
\ No newline at end of file
+public class PencilGame {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static final Random random = new Random();
+    public static int pencilsTaken;
+
+    public static int getNumberOfPencils() {
```

Comment:
You have a lot of static methods, yet you also have a state (which are static as well, such as number of pencils taken, random and the scanner), yet you also create a new game every time the game is started.

This means that each game that is started in the same JVM is sharing this state. Is that what you intended? Can you think of the reason why you want every method and state to be static (and therefore shared between each and every instance of the class)?